package.json

```diff
@@ -32,12 +32,14 @@
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
+    "browserify": "^13.1.1",
+    "tsify": "^2.0.7",
     "@types/pixi.js": "^4.3.1",
     "typescript": "^2.1.4"
   },
   "scripts": {
     "demo": "npm run build && open demo/index.html",
-    "build": "tsc",
+    "build": "tsc; browserify pixi-multistyle-text.ts -p tsify -s MultiStyleText -o dist/pixi-multistyle-text.js -d",
```

Comment:
The point of Browserify here is to export the module for CommonJS and AMD if either of those are being used, and otherwise defaulting to a global export.  This is similar to how Pixi itself functions (and they also use Browserify to build).  I was originally hoping we could get away with using TS's `umd` module mode, which handles both CommonJS and AMD properly, but it doesn't default to exposing the module globally if neither is found.

I'd also like to remove the "double wrapping" problem if possible, but I couldn't find a way to do so.  (In particular, I would've expected that `export default class MultiStyleText` would do the trick, but that instead exports the class to `MultiStyleText.default`.)

I'll keep looking into it.